packages/components/src/components/desktop-wrapper/desktop-wrapper.tsx

```diff
@@ -0,0 +1,13 @@
+import { ReactNode } from 'react';
+import { isDesktop } from '@deriv/shared';
+
+type TDesktopProps = {
+    children: ReactNode;
```

Comment:
Better use React.ReactNode since we get no named imports error from webpack when we use named imports from react.

Reply:
Changed.